### ISSUE TITLE
Rotated animated elements are rendered at the wrong size

### ISSUE THREAD

**Issue by troywarr** [niklasvh/html2canvas#1559]
### Bug report:

HTML elements whose `rotate()` property is animated via CSS are rendered at the wrong size if the value of `rotate()` is something other than 0.

Please see this example:

https://codepen.io/troywarr/pen/GGmKrp

Each of the 3 rows contains an HTML element (a `<div>` with a child `<p>`) on the left and its corresponding `<canvas>` (rendered via html2canvas) on the right.

The first row shows the `<p>` rotated using a CSS transform, and html2canvas renders it exactly as intended:

```css
.transformed.working p {
  transform: rotate(45deg);
}
```

The second row shows the `<p>` rotated 0 degrees using a CSS animation, and html2canvas also renders it exactly as intended:

```css
.animated.working p {
  animation-name: rotate0; /* rotates 0 degrees */
  animation-duration: 1ms;
  animation-play-state: paused;  
}
```

The third row shows the `<p>` rotated 45 degrees using a CSS animation:

```css
.animated.broken p {
  animation-name: rotate45; /* rotates 45 degrees */
  animation-duration: 1ms;
  animation-play-state: paused;
}
```

In this case, html2canvas does __not__ render it as intended. In the rendered `<canvas>`, the `<p>` is too large and is shifted downwards and leftwards from its center point. The text within the `<p>` is also not centered vertically, apparently because its CSS `line-height` is now smaller than its oversized parent `<div>`.

### Specifications:

 * html2canvas version tested with: v1.0.0-alpha.12
 * Browser & version: Google Chrome 66.0.3359.181
 * Operating system: macOS High Sierra 10.13.4

**Comment by StarStep** [comment]
Does not work with transitions too.
Updated demo: https://codepen.io/StarStep/pen/jpaQYp

**Comment by StefanWokusch** [comment]
I have the problem also with transitions. I'm using the library for automated testing. The main problem i have with that bug is, that it creates each time a different capture.

**Comment by nidi3** [comment]
a fixed version is available at `"@nidi/html2canvas": "1.0.0-rc.13"`
